_includes/getting-started.html

```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
             </code></pre>
+          </div> -->
+          <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+            <div style="width:auto; height:30px; padding:0px; position:relative;">
+              <pre><code>   $ git clone https://github.com/layer5io/meshery.git; cd meshery    
+              </code></pre>
+              <!-- copy to clipboard -->
+              <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                    cursor: pointer; background:#00a4ca; width:60px; height:44px; margin-bottom:35px; 
+                    border-radius: 2px; margin:0px 0px 880px 0px;"
+                data-clipboard-target="#start"
+                data-clipboard-text="git clone https://github.com/layer5io/meshery.git; cd meshery    ">
+                <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+              </a>
           </div>
+        </div>
         <p>Install Meshery on your cluster by running the following:</p>
-        <div class="highlight-code highlight-source-shell">
+        <!-- <div class="highlight-code highlight-source-shell">
           <pre><code>
   $ kubectl create ns meshery
   $ kubectl -n meshery apply -f deployment_yamls/k8s
           </code></pre>
-          <!-- PROVIDE THE "ON THE MESH" VERSION IN FUTURE DOCUMENTATION
-          # additional step for running on Istio
-          $ kubectl -n meshery apply -f deployment_yamls/istio.yaml -->
+        </div> -->
+        <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+          <div style="width:auto; height:60px; padding:0px; position:relative;">
+            <pre><code> $ git clone https://github.com/layer5io/meshery.git; cd meshery<br /> $ kubectl create namespace meshery<br /> $ helm template meshery --namespace meshery install/kubernetes/helm/meshery<br /> 
+            </code></pre>
+            <!-- copy to clipboard -->
+            <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+              style="position: absolute;top:-14px; right: -1px; border: 0; 
+                  cursor: pointer; background:#00a4ca; width:60px; height:74px; margin-bottom:35px; 
+                  border-radius: 2px; margin:0px 0px 880px 0px;"
+              data-clipboard-target="#start"
+              data-clipboard-text="kubectl create ns meshery
+              kubectl -n meshery apply -f deployment_yamls/k8s ">
+              <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+            </a>
         </div>
       </div>
+      </div>
     
     <h3 style="margin-left:1em;">Windows user</h3> 
     <p style="margin-left:1.8em;display: inline;">
```

Comment:
Will you change Windows from a hyperlink to the documentation to including the download binary instructions directly here? This way, the user doesn't need to be directed to another site for these instructions.

Reply:
sir I have added